docs/Installation.md

```diff
@@ -49,6 +49,8 @@ Check the options here if you know what you're doing: [CMake options](CMake-opti
 
 At this point, you must be in your "build/" directory.
 
+**Note**: in the follows command the variable `$HOME` is the path of the **current user**, so if you are logged as root, $HOME will be "/root".
```

Comment:
in the following command, the environment variable $HOME points towards your current user home directory (/home/my_user or /root if you are logged as root). Try it `echo $HOME`.

@Helias